pkg/fanal/analyzer/secret/secret.go

```diff
@@ -63,6 +66,10 @@ func init() {
 	analyzer.RegisterAnalyzer(NewSecretAnalyzer(secret.Scanner{}, ""))
 }
 
+func isAllowedBinary(filename string) bool {
```

Comment:
What if we omit the `is` prefix?
```suggestion
func allowedBinary(filename string) bool {
```

Reply:
done